### DISCUSSION TITLE
Do native iOS crashes get reported?

### DISCUSSION THREAD

**Discussion by thipokch** [firebase/flutterfire#9087] [Q&A]
Do native iOS crashes get reported to Firebase Crashlytics when FlutterFire is setup? If not, how would I go about setting it up, so that there won't be duplicates? Thanks.

**Comment by a1573595** [comment]
The same question, Doc wrote Android has optional, nothing for iOS.

https://firebase.flutter.dev/docs/crashlytics/overview/

Do you know the answer now? @thipokch

**Reply by thipokch** [reply]
Yes it does.

**Reply by a1573595** [reply]
Thanks.
